CO_high_level: 3000
CNG_high_level: 2048
LPG_high_level: 2048
H_high_level: 2048
pan_max_position: 3595
pan_min_position: 500
pan_neutral_position: 2048
pan_position_scale: 0.00306796157
pan_motor_reverse: false
pan_potentiometer_reverse: true
pan_output_torque_per_amp: 1.0
tilt_max_position: 4095
tilt_min_position: 0
tilt_neutral_position: 2048
tilt_position_scale: 0.0
tilt_motor_reverse: false
tilt_potentiometer_reverse: false
tilt_ouput_torque_per_amp: 1.0
deploy_max_position: 4096
deploy_min_position: 0
deploy_neutral_position: 2048
deploy_position_scale: 0.0
deploy_motor_reverse: false
deploy_potentiometer_reverse: false
motor_output_power_scale: 255
motor_output_auto_enable: true
